src/common/Utilities/Random.cpp

```diff
@@ -17,13 +17,76 @@
 
 #include "Random.h"
 #include "Errors.h"
+
+#if defined( RASPBERRY_PI )
+#include "../mersennetwister/MersenneTwister.h"
```

Comment:
Is there a reason you add external mersenne library instead of using the one in std::?

Reply:
Good idea, i will gonna do that.